pkg/action/rke_test.go

```diff
@@ -0,0 +1,51 @@
+package action_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/teslamotors/vehicle-command/pkg/action"
+	"github.com/teslamotors/vehicle-command/pkg/protocol/protobuf/vcsec"
+)
+
+var _ = Describe("RKE Actions", func() {
+	Describe("AutoSecureVehicle", func() {
+		It("returns auto secure vehicle action", func() {
+			action := action.AutoSecureVehicle()
+			Expect(action).ToNot(BeNil())
+			Expect(action.SubMessage).ToNot(BeNil())
+			Expect(action.SubMessage).To(BeAssignableToTypeOf(&vcsec.UnsignedMessage_RKEAction{}))
+			Expect(action.SubMessage.(*vcsec.UnsignedMessage_RKEAction).RKEAction).To(Equal(vcsec.RKEAction_E_RKE_ACTION_AUTO_SECURE_VEHICLE))
```

Comment:
The Golang protobuf code checks for nil deferences and returns default values, so I think you could just do:

```go
action := action.AutoSecureVehicle()
Expect(action.GetRKEAction()).To(Equal(vcsec.RKEAction_E_RKE_ACTION_AUTO_SECURE_VEHICLE))
```

Note that this wouldn't work for `RKEAction_E_RKE_ACTION_UNLOCK`, since the protobuf definition breaks the convention of reserving the 0 (default) value for UNKNOWN.